src/jobs/build_push_image.yml

```diff
@@ -90,5 +93,14 @@ steps:
             name: restore the dependencies cache
             key: <<parameters.cache_name>>
   - steps: << parameters.prebuild_steps >>
-  - steps: << parameters.build_steps >>
+  - when:
+      condition: <<parameters.working_directory>>
+      steps:
+        - run:
+            name: Build Docker image
+            command: isopod build
+            working_directory: <<parameters.working_directory>>
```

Comment:
This as well can be separate command with working dir parameter. As I suggested for `quality_gate` command. Then, it can be reused for default values as well.